docs/get-started/create-service-principal.md

```diff
@@ -216,15 +216,31 @@ data:        NotActions:   Microsoft.Authorization/*/Write,Microsoft.Authorizati
 ```
 
 ## 2.5 Verify Your Service Principal
-After your service principal is created, please verify it with the following command:
 
-### 2.5.1 Login With your Service Principal
+Your service principal is created as follows:
 
-```
-azure login --username <CLIENT-ID> --password <CLIENT-SECRET> --service-principal --tenant <TENANT-ID>
-```
+- **TENANT-ID**
+- **CLIENT-ID**
+- **CLIENT-SECRET** 
 
-Example:
-```
-azure login -u 246e4af7-75b5-494a-89b5-363addb9f0fa -p "password" --service-principal --tenant 22222222-1234-5678-1234-678912345678
-```
\ No newline at end of file
+Please verify it with the following steps:
+
+1. Use Azure CLI to login with your service principal.
+
+  You can find the `TENANT-ID`, `CLIENT-ID`, and `CLIENT-SECRET` properties in the `~/bosh.yml` file. If you cannot login, then the service principal is invalid.
+
+  ```
+  azure login --username <CLIENT-ID> --password <CLIENT-SECRET> --service-principal --tenant <TENANT-ID>
+  ```
+
+  Example:
+
+  ```
+  azure login -u 246e4af7-75b5-494a-89b5-363addb9f0fa -p "password" --service-principal --tenant 22222222-1234-5678-1234-678912345678
```

Comment:
Could you help to change the abbreviation to full name for all CLI parameter names in all documents?
azure login --username 246e4af7-75b5-494a-89b5-363addb9f0fa --password "password" --service-principal --tenant 22222222-1234-5678-1234-678912345678